### ISSUE TITLE
Time and Battery display

### ISSUE THREAD

**Issue by siamak01** [lhc70000/iina#1473]
**System and IINA version:**

- macOS 10.13.2
- IINA 0.0.15

**Expected behavior:**
I do not know how to report for a bug and new feature. So I am here.
On Display time does not come back by hovering the mouse arrow. It reappears only after pause/play or fullscreen/maximaze/minimazie.

**Actual behavior:**
Requested feature which I badly want and none of other players have as far as I can say is:
Crossfade for Audio(iTunes does have) But no player has have Video crossfade.
I like to set a time in second for crossfade for audio an video play.

**Comment by inflation** [comment]
By time display, are you suggesting the controller?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by siamak01** [comment]
I was trying to check the Preferences. "On Screen Display" has check box
for time and battery info. I assumed it will display time of the video on
full screen. It will show on X.XX number on upper right  corner. I am using
only desktop mac not laptop.  Maybe this is to do with the life of laptop
battery. If it is, then it is o.k. and only may be should read " Display
battery time and info" if intended to display only battery life time left".

**Comment by lhc70000** [comment]
> Crossfade for Audio(iTunes does have) But no player has have Video crossfade.

It's not possible in mpv, and so does IINA.

**Comment by megadr01d** [comment]
Any special reason not to include the elapsed/remaining time in [this information](https://github.com/lhc70000/iina/issues/179) and, optionally **always** display it, while in fullscreen (don't hide it with the controller)?

Also, battery info should be a separate option (I run a laptop without a battery so I always get a useless 0%).

**Comment by lhc70000** [comment]
@megadr01d Please open a new issue for this.
